index.bs

```diff
@@ -102,22 +102,18 @@ configuration. The methods come in two by two flavours:
 ## Sanitizer API ## {#sanitizer-api}
 
 The {{Element}} interface defines two methods, {{Element/setHTML()}} and
-{{Element/setHTMLUnsafe__TO_BE_MERGED()}}. Both of these take a {{DOMString}} with HTML
+{{Element/setHTMLUnsafe()}}. Both of these take a {{DOMString}} with HTML
 markup, and an optional configuration.
 
-<pre class=idl>
+<pre class="idl extract">
```

Comment:
This will also prevent `setHTML` from being extracted. Is that OK, should they go together?

Reply:
Yeah, that seems fine.

Reply:
My thinking was that a "partial" integration wouldn't help us much and that we'd simply take out all the bits where we extend interfaces defined in HTML.

Reply:
OK, that seems OK. LGTM!